dash.el

```diff
@@ -65,7 +65,9 @@ special values."
   `(setq ,list (cdr ,list)))
 
 (defmacro --each (list &rest body)
-  "Anaphoric form of `-each'."
+  "Eval BODY with symbol `it' bound to each element of the LIST.
+Return nil, used for side-effects only.
+Anaphoric form of `-each'."
```

Comment:
I think a bit cleaner (from the perspective of reading the source code) way would be to put an empty line after the first line and before the "reference link" on the last.

``` elisp
"Eval BODY with symbol `it' bound to each element of the LIST.

Return nil, used for side-effects only.

Anaphoric form of `-each'."
```

This draws the attention to the first line better which is the quick summary.  On some functions we also say `See also: ....` which also is on its own line so it would be a bit more consistent.

WDYT?

Reply:
I agree that it's clearer with extra spaces. I like it better also. The question is if it doesn't add too much white-noise and if it's normal to have extra whitespace all over the place in the doc-strings.

Reply:
I think it's pretty normal in Elisp world.  On some of my projects I have docstrings spaning multiple pages: https://github.com/Fuco1/smartparens/blob/9738360eb2afb58b4c21815f9d5c793b8125f540/smartparens.el#L2292